psl-core/src/main/java/org/linqs/psl/database/rdbms/RDBMSDatabase.java

```diff
@@ -18,22 +18,18 @@
 package org.linqs.psl.database.rdbms;
 
 import org.linqs.psl.config.Options;
-import org.linqs.psl.database.DataStore;
 import org.linqs.psl.database.Database;
 import org.linqs.psl.database.DatabaseQuery;
 import org.linqs.psl.database.Partition;
 import org.linqs.psl.database.ResultList;
 import org.linqs.psl.database.QueryResultIterable;
-import org.linqs.psl.database.atom.AtomCache;
 import org.linqs.psl.model.atom.GroundAtom;
-import org.linqs.psl.model.atom.ObservedAtom;
 import org.linqs.psl.model.atom.QueryAtom;
 import org.linqs.psl.model.atom.RandomVariableAtom;
```

Comment:
Can this import also be dropped?

Reply:
Yes this is dropped in online PSL branch as well, if that is what you are asking?